new file:   .gitignore

NastyaPilat opened the pull request:
new file:   allure_results/05115efc-012e-4a01-8c88-278bc8708692-container.json
	new file:   allure_results/1095d631-588b-4104-a3e6-0bd7306642d7-result.json
	new file:   allure_results/14718f87-97cb-4879-8b84-30f20ce94402-result.json
	new file:   allure_results/1484a19d-b965-45b7-986a-7ce1aca54ec9-container.json
	new file:   allure_results/1ab4064a-3686-4911-9fb0-c8ab8595c4a6-container.json
	new file:   allure_results/26f3b6d6-0357-4df5-aa59-687d84172990-container.json
	new file:   allure_results/2adefcca-0854-4673-926a-9c122637998c-container.json
	new file:   allure_results/2cb7c6b6-916f-457d-af7c-46bd2b9a2543-container.json
	new file:   allure_results/36c5c012-240f-40a5-9d09-31d15f92addd-container.json
	new file:   allure_results/3a36feb4-4ef5-4d93-bdce-cf23f5571f2f-result.json
	new file:   allure_results/486c149e-abe9-4432-90b2-d34836164dc5-container.json
	new file:   allure_results/4c846a62-469d-49e0-aea0-5a57abd9d890-result.json
	new file:   allure_results/58cbda00-2402-4081-8535-2b3a9bfa1468-result.json
	new file:   allure_results/5954a060-01d9-4832-a6a6-52fc7754c4a3-result.json
	new file:   allure_results/6060ab02-3731-4712-b305-06bbee911f5c-result.json
	new file:   allure_results/631378a9-d74b-4794-8bf1-a83934c17a70-container.json
	new file:   allure_results/6a4ffbd4-8664-4ddf-a518-01d38bc898b7-result.json
	new file:   allure_results/6aa37222-3f29-4b1a-8178-a4c91a878646-container.json
	new file:   allure_results/73adc49e-1237-49b6-a291-2534168bdb7a-result.json
	new file:   allure_results/76c28eb2-d8e1-4bc0-84c0-63866920c108-container.json
	new file:   allure_results/8779da82-c620-4063-99ac-c7c6f91be031-result.json
	new file:   allure_results/87db3f8f-d177-44c5-8f06-0c25534de327-container.json
	new file:   allure_results/929d503d-ebea-413f-bcc5-8127b363f1d9-container.json
	new file:   allure_results/a79f9304-45eb-4007-8be7-9a7d5f91e617-container.json
	new file:   allure_results/b0c260b9-b777-4beb-a3e6-70fdb218f691-result.json
	new file:   allure_results/bbc1c569-2532-488c-8be6-f7a7ffea4a1b-container.json
	new file:   allure_results/bf27a099-437d-4f40-9ce7-a36f8d4bf878-container.json
	new file:   allure_results/c0952c72-ac1e-47f0-94b0-01ac05d7af3e-result.json
	new file:   allure_results/c89513dd-5a2d-4ab4-8904-cc0f5b5b281b-container.json
	new file:   allure_results/cae12601-cf82-4ee1-83b9-6cc27283a99a-container.json
	new file:   allure_results/d331d95a-6509-4f47-85a0-5163a5fb267c-container.json
	new file:   conftest.py
	new file:   pages/base_page.py
	new file:   pages/home_page.py
	new file:   pages/order_page.py
	new file:   requirements.txt
	new file:   tests/test_base_page.py
	new file:   tests/test_home_page.py
	new file:   tests/test_order_page.py

maxpilat committed:
new file:   .gitignore
	new file:   allure_results/05115efc-012e-4a01-8c88-278bc8708692-container.json
	new file:   allure_results/1095d631-588b-4104-a3e6-0bd7306642d7-result.json
	new file:   allure_results/14718f87-97cb-4879-8b84-30f20ce94402-result.json
	new file:   allure_results/1484a19d-b965-45b7-986a-7ce1aca54ec9-container.json
	new file:   allure_results/1ab4064a-3686-4911-9fb0-c8ab8595c4a6-container.json
	new file:   allure_results/26f3b6d6-0357-4df5-aa59-687d84172990-container.json
	new file:   allure_results/2adefcca-0854-4673-926a-9c122637998c-container.json
	new file:   allure_results/2cb7c6b6-916f-457d-af7c-46bd2b9a2543-container.json
	new file:   allure_results/36c5c012-240f-40a5-9d09-31d15f92addd-container.json
	new file:   allure_results/3a36feb4-4ef5-4d93-bdce-cf23f5571f2f-result.json
	new file:   allure_results/486c149e-abe9-4432-90b2-d34836164dc5-container.json
	new file:   allure_results/4c846a62-469d-49e0-aea0-5a57abd9d890-result.json
	new file:   allure_results/58cbda00-2402-4081-8535-2b3a9bfa1468-result.json
	new file:   allure_results/5954a060-01d9-4832-a6a6-52fc7754c4a3-result.json
	new file:   allure_results/6060ab02-3731-4712-b305-06bbee911f5c-result.json
	new file:   allure_results/631378a9-d74b-4794-8bf1-a83934c17a70-container.json
	new file:   allure_results/6a4ffbd4-8664-4ddf-a518-01d38bc898b7-result.json
	new file:   allure_results/6aa37222-3f29-4b1a-8178-a4c91a878646-container.json
	new file:   allure_results/73adc49e-1237-49b6-a291-2534168bdb7a-result.json
	new file:   allure_results/76c28eb2-d8e1-4bc0-84c0-63866920c108-container.json
	new file:   allure_results/8779da82-c620-4063-99ac-c7c6f91be031-result.json
	new file:   allure_results/87db3f8f-d177-44c5-8f06-0c25534de327-container.json
	new file:   allure_results/929d503d-ebea-413f-bcc5-8127b363f1d9-container.json
	new file:   allure_results/a79f9304-45eb-4007-8be7-9a7d5f91e617-container.json
	new file:   allure_results/b0c260b9-b777-4beb-a3e6-70fdb218f691-result.json
	new file:   allure_results/bbc1c569-2532-488c-8be6-f7a7ffea4a1b-container.json
	new file:   allure_results/bf27a099-437d-4f40-9ce7-a36f8d4bf878-container.json
	new file:   allure_results/c0952c72-ac1e-47f0-94b0-01ac05d7af3e-result.json
	new file:   allure_results/c89513dd-5a2d-4ab4-8904-cc0f5b5b281b-container.json
	new file:   allure_results/cae12601-cf82-4ee1-83b9-6cc27283a99a-container.json
	new file:   allure_results/d331d95a-6509-4f47-85a0-5163a5fb267c-container.json
	new file:   conftest.py
	new file:   pages/base_page.py
	new file:   pages/home_page.py
	new file:   pages/order_page.py
	new file:   requirements.txt
	new file:   tests/test_base_page.py
	new file:   tests/test_home_page.py
	new file:   tests/test_order_page.py

AlexKitIt commented on pages/base_page.py:
```diff
@@ -0,0 +1,28 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+
+
+class BasePage:
```
Нужно исправить: Объект страницы - это одна конкретная страница, допустим https://qa-scooter.praktikum-services.ru/ на ней есть кнопка заказа, ссылка для перехода на дзен и вопросы. 
Второй объект - это страница https://qa-scooter.praktikum-services.ru/orders и тут уже наши поля для заполнения заказа. А в этих объектах мы храним конкретные элементы и описываем действие с ними. 
Кроме того в работе просто остутствует валидный BasePage, в котором будут базовые методы для взаимодействия со всеми элементами по типу клика или send_keys и базовые wait, которых уже не должно быть в объектах страниц, 
там, за счет наследования объектов от Base, можно будет эти общие методы вызывать уже для элементов страниц. Вот эта модель и будет PageObject. 
Все обращения к driver - в BasePage, все работа с конкретными элементами на страницах - в объектах страниц

AlexKitIt commented on pages/home_page.py:
```diff
@@ -0,0 +1,34 @@
+    def __init__(self, driver: WebDriver):
+        self.driver = driver
+
+    def get_question(self, index: int):
```
Нужно исправить: здесь и далее: над каждым методом пейджи ставится аннотация allure.step

AlexKitIt commented on pages/home_page.py:
```diff
@@ -0,0 +1,34 @@
+        self.driver = driver
+
+    def get_question(self, index: int):
+        return self.driver.find_elements(*self.question)[index]
```
Нужно исправить: здесь и далее по проекту: все прямые обращения к driver и waitы необходимо вынести в base и оттуда за счет наследования переиспользовать в объектах страниц

AlexKitIt commented on pages/home_page.py:
```diff
@@ -0,0 +1,34 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+
+
+class HomePage:
```
Нужно исправить: здесь и далее: все пейджи наследуются от base

AlexKitIt commented on pages/order_page.py:
```diff
@@ -0,0 +1,65 @@
+        self.driver.find_element(*self.input_second_name).send_keys(second_name)
+        self.driver.find_element(*self.input_address).send_keys(address)
+        self.driver.find_element(*self.input_metro_station).click()
+        self.driver.find_element(By.XPATH, f"//button[contains(div[@class='Order_Text__2broi'], '{metro_station}')]").click()
```
Нужно исправить: здесь и далее: все локаторы хранятся отдельно от методов, даже этот темпелйт

AlexKitIt commented on tests/test_base_page.py:
```diff
@@ -0,0 +1,31 @@
+
+
+class TestBasePage:
+    driver = None
+
+    @classmethod
+    def setup_class(cls):
+        cls.driver = webdriver.Chrome()
```
Можно лучше: лучше использовать фикстуры, они значительно упростят жизнь

AlexKitIt commented on tests/test_base_page.py:
```diff
@@ -0,0 +1,31 @@
+    def setup_class(cls):
+        cls.driver = webdriver.Chrome()
+
+    def test_logo_yandex(self):
```
Нужно исправить: здесь и далее: над каждым тестовым методом ставится аннотация allure.title

AlexKitIt commented on tests/test_base_page.py:
```diff
@@ -0,0 +1,31 @@
+        cls.driver = webdriver.Chrome()
+
+    def test_logo_yandex(self):
+        self.driver.get("https://qa-scooter.praktikum-services.ru/")
```
Можно лучше: урлы стоит хранить отдельно, они постоянно переиспользуются

AlexKitIt commented on tests/test_base_page.py:
```diff
@@ -0,0 +1,31 @@
+
+    def test_logo_yandex(self):
+        self.driver.get("https://qa-scooter.praktikum-services.ru/")
+        base_page = BasePage(self.driver)
```
Нужно исправить: объект base никогда не создается, это класс интерфейс

AlexKitIt commented on tests/test_base_page.py:
```diff
@@ -0,0 +1,31 @@
+        self.driver.get("https://qa-scooter.praktikum-services.ru/")
+        base_page = BasePage(self.driver)
+        base_page.click_logo_yandex()
+        WebDriverWait(self.driver, 3).until(EC.number_of_windows_to_be(2))
```
Нужно исправить: здесь и далее: прямого обращения к wait из тестов быть не должно, только через методы пейджей

AlexKitIt commented on tests/test_base_page.py:
```diff
@@ -0,0 +1,31 @@
+        base_page = BasePage(self.driver)
+        base_page.click_logo_yandex()
+        WebDriverWait(self.driver, 3).until(EC.number_of_windows_to_be(2))
+        self.driver.switch_to.window(self.driver.window_handles[1])
```
Нужно исправить: в base

AlexKitIt commented on tests/test_home_page.py:
```diff
@@ -0,0 +1,63 @@
+    def setup_class(cls):
+        cls.driver = webdriver.Chrome()
+
+    def test_question_1(self):
```
Нужно исправить: тест на вопросы должен быть один параметризованный(на вход примет элемент вопроса, элемент ответа и текст ответа)

AlexKitIt commented on tests/test_home_page.py:
```diff
@@ -0,0 +1,63 @@
+        self.driver.get('https://qa-scooter.praktikum-services.ru/')
+        BasePage.accept_cookies(self.driver)
+        home_page = HomePage(self.driver)
+        assert home_page.check_question(0, 'Сутки — 400 рублей. Оплата курьеру — наличными или картой.')
```
Можно лучше: тексты ответов стоит хранить снаружи в data модулях для поддержания и актуализации

maxpilat committed:
new file:   allure_results/00a584b0-bfa7-4365-81f2-b26157278a0f-container.json
	new file:   allure_results/0489121d-ad0b-4a41-a27d-eaf7a9567a41-container.json
	deleted:    allure_results/05115efc-012e-4a01-8c88-278bc8708692-container.json
	new file:   allure_results/06004cfb-77aa-4a8f-a353-ebfd15e165e4-container.json
	new file:   allure_results/0b220ce4-6ece-461c-a143-1f75c39ebeed-container.json
	new file:   allure_results/0c2cf39b-3024-43a1-9084-a06f53298f0d-result.json
	deleted:    allure_results/1095d631-588b-4104-a3e6-0bd7306642d7-result.json
	new file:   allure_results/13b020c7-1450-483b-85c0-988b58209ae0-result.json
	deleted:    allure_results/14718f87-97cb-4879-8b84-30f20ce94402-result.json
	deleted:    allure_results/1484a19d-b965-45b7-986a-7ce1aca54ec9-container.json
	new file:   allure_results/156fa906-6cdb-40c9-a9cb-1876d8b31bd0-container.json
	new file:   allure_results/15ccf316-0420-4b3a-901d-eb5ee6da7527-container.json
	deleted:    allure_results/1ab4064a-3686-4911-9fb0-c8ab8595c4a6-container.json
	new file:   allure_results/1ad71a2f-4556-432b-8580-34b3d5ad2b26-result.json
	new file:   allure_results/1ef96be7-358e-4d33-8c37-fa7f256a2f5b-container.json
	new file:   allure_results/206c9c7e-7f75-4bbb-aeaa-e7a5d1b293e9-container.json
	new file:   allure_results/25a4c4ca-e2d8-4aed-a593-d0fa84ec6f32-container.json
	new file:   allure_results/26690e3c-03a5-4939-8010-c619f4520455-container.json
	deleted:    allure_results/26f3b6d6-0357-4df5-aa59-687d84172990-container.json
	new file:   allure_results/277a2a9a-b1de-406f-9c5c-877f77e8eef7-container.json
	deleted:    allure_results/2adefcca-0854-4673-926a-9c122637998c-container.json
	new file:   allure_results/2b0aec79-f3bf-4f46-8ffc-40af9b28071f-container.json
	new file:   allure_results/2ba70434-0b59-4648-ac70-81d6462ab8bc-container.json
	deleted:    allure_results/2cb7c6b6-916f-457d-af7c-46bd2b9a2543-container.json
	new file:   allure_results/3650e919-fb24-4a35-b852-f358b0eb0c74-container.json
	deleted:    allure_results/36c5c012-240f-40a5-9d09-31d15f92addd-container.json
	deleted:    allure_results/3a36feb4-4ef5-4d93-bdce-cf23f5571f2f-result.json
	new file:   allure_results/473963c8-cda2-4e1b-b75d-b2460a23b0d6-container.json
	deleted:    allure_results/486c149e-abe9-4432-90b2-d34836164dc5-container.json
	deleted:    allure_results/4c846a62-469d-49e0-aea0-5a57abd9d890-result.json
	new file:   allure_results/553fb0c9-7a62-48b4-b68a-e8e2232ba277-container.json
	deleted:    allure_results/58cbda00-2402-4081-8535-2b3a9bfa1468-result.json
	deleted:    allure_results/5954a060-01d9-4832-a6a6-52fc7754c4a3-result.json
	new file:   allure_results/5c27012f-6830-4563-acb0-274adf91c39a-container.json
	new file:   allure_results/5f154c07-7c05-456f-913f-7d2372722bd0-result.json
	deleted:    allure_results/6060ab02-3731-4712-b305-06bbee911f5c-result.json
	new file:   allure_results/61b6a131-22c7-4820-bafb-48be65b70fc3-container.json
	deleted:    allure_results/631378a9-d74b-4794-8bf1-a83934c17a70-container.json
	new file:   allure_results/650bc7e7-ad1f-4f4a-a7b1-85199b5f6cfe-container.json
	new file:   allure_results/6a21acc8-3ccf-4775-80c0-4dcfe44dc396-result.json
	deleted:    allure_results/6a4ffbd4-8664-4ddf-a518-01d38bc898b7-result.json
	deleted:    allure_results/6aa37222-3f29-4b1a-8178-a4c91a878646-container.json
	new file:   allure_results/6b782ec5-4a3a-43c1-a7b8-3bb19d06d50c-result.json
	deleted:    allure_results/73adc49e-1237-49b6-a291-2534168bdb7a-result.json
	deleted:    allure_results/76c28eb2-d8e1-4bc0-84c0-63866920c108-container.json
	new file:   allure_results/7865499d-0716-4e16-a9a7-aa86339955af-container.json
	new file:   allure_results/791a7738-fdb6-4bda-b76b-94a0c5f6aac8-container.json
	new file:   allure_results/7b3ba24a-f128-45f2-816b-ab50c7626999-container.json
	deleted:    allure_results/8779da82-c620-4063-99ac-c7c6f91be031-result.json
	deleted:    allure_results/87db3f8f-d177-44c5-8f06-0c25534de327-container.json
	deleted:    allure_results/929d503d-ebea-413f-bcc5-8127b363f1d9-container.json
	new file:   allure_results/9862a721-42fe-4cc9-9848-b0714bc21991-container.json
	new file:   allure_results/99ce585c-5f1a-46c9-aa65-45117ccd630e-result.json
	new file:   allure_results/9c70da65-e3fc-4ffa-bf40-49d8dc25784f-container.json
	new file:   allure_results/a6204d5a-a252-43db-88c2-0c65c40f7b89-result.json
	deleted:    allure_results/a79f9304-45eb-4007-8be7-9a7d5f91e617-container.json
	deleted:    allure_results/b0c260b9-b777-4beb-a3e6-70fdb218f691-result.json
	new file:   allure_results/b15a3024-96ab-4084-8469-3df26e788a86-container.json
	new file:   allure_results/b4e63225-55e7-434b-a0fb-d1a594a11f75-result.json
	deleted:    allure_results/bbc1c569-2532-488c-8be6-f7a7ffea4a1b-container.json
	new file:   allure_results/bdff784f-6bd5-463d-8f98-f0742ef563fe-container.json
	deleted:    allure_results/bf27a099-437d-4f40-9ce7-a36f8d4bf878-container.json
	deleted:    allure_results/c0952c72-ac1e-47f0-94b0-01ac05d7af3e-result.json
	new file:   allure_results/c4303dca-2cec-40ef-88a4-635351c90359-container.json
	deleted:    allure_results/c89513dd-5a2d-4ab4-8904-cc0f5b5b281b-container.json
	new file:   allure_results/c8bf5c30-b9de-45de-b262-2d65bf5330a6-container.json
	deleted:    allure_results/cae12601-cf82-4ee1-83b9-6cc27283a99a-container.json
	new file:   allure_results/cc3e1048-3d0c-49d4-9a76-fd0c06cfd7c7-container.json
	deleted:    allure_results/d331d95a-6509-4f47-85a0-5163a5fb267c-container.json
	new file:   allure_results/d35644c7-801c-48b8-8a36-50b592d7edd3-container.json
	new file:   allure_results/e0b6921e-6c8c-469e-8056-baa129161c0a-result.json
	new file:   allure_results/ef784b5d-1877-4458-a070-1df970281138-container.json
	new file:   allure_results/f471ccee-04db-4c5a-abd1-7a03a8a561f9-container.json
	new file:   allure_results/f50b98ae-7b2d-4c1e-8f66-b2b79bef3c28-container.json
	new file:   allure_results/fa5a6bc4-cff9-4676-877a-04fa5a97a2bd-container.json
	new file:   allure_results/fadc9b19-df43-4f5b-abd1-f2ddc72f9c9f-result.json
	new file:   allure_results/feb7ec5b-9062-4729-b281-596ad16134e1-container.json
	new file:   allure_results/ff0c27ec-5f56-4144-ac94-6726df8ec899-result.json
	modified:   conftest.py
	new file:   data.py
	new file:   locators/base_page_locators.py
	new file:   locators/home_page_locators.py
	new file:   locators/order_page_locators.py
	modified:   pages/base_page.py
	modified:   pages/home_page.py
	modified:   pages/order_page.py
	modified:   tests/test_base_page.py
	modified:   tests/test_home_page.py
	modified:   tests/test_order_page.py

AlexKitIt commented on pages/home_page.py:
```diff
@@ -0,0 +1,38 @@
+
+class HomePage(BasePage):
+
+    @allure.step
```
Нужно исправить: здесь и далее: этот декоратор принимает название шага

AlexKitIt commented on tests/test_base_page.py:
```diff
@@ -0,0 +1,24 @@
+        home_page.click_logo_yandex()
+        home_page.wait_for_number_of_windows_to_be(2)
+        home_page.switch_to_window(1)
+        assert data.DZEN_PAGE_URL == driver.current_url
```
Повторный комент: Нужно исправить: здесь и далее: прямого обращения к driver из тестов быть не должно, только через методы пейджей

maxpilat committed:
modified:   conftest.py
	modified:   data.py
	modified:   locators/base_page_locators.py
	modified:   locators/home_page_locators.py
	modified:   locators/order_page_locators.py
	modified:   pages/base_page.py
	modified:   pages/home_page.py
	modified:   pages/order_page.py
	modified:   requirements.txt
	modified:   tests/test_base_page.py
	modified:   tests/test_home_page.py
	modified:   tests/test_order_page.py

maxpilat committed:
deleted:    allure_results/00a584b0-bfa7-4365-81f2-b26157278a0f-container.json
	deleted:    allure_results/0489121d-ad0b-4a41-a27d-eaf7a9567a41-container.json
	deleted:    allure_results/06004cfb-77aa-4a8f-a353-ebfd15e165e4-container.json
	deleted:    allure_results/0b220ce4-6ece-461c-a143-1f75c39ebeed-container.json
	deleted:    allure_results/0c2cf39b-3024-43a1-9084-a06f53298f0d-result.json
	deleted:    allure_results/13b020c7-1450-483b-85c0-988b58209ae0-result.json
	new file:   allure_results/146756c8-6f44-4dda-9a3f-524b1fe2388e-container.json
	deleted:    allure_results/156fa906-6cdb-40c9-a9cb-1876d8b31bd0-container.json
	deleted:    allure_results/15ccf316-0420-4b3a-901d-eb5ee6da7527-container.json
	deleted:    allure_results/1ad71a2f-4556-432b-8580-34b3d5ad2b26-result.json
	new file:   allure_results/1e6ab728-b5e8-4bef-b3b8-805f399ab7bf-container.json
	deleted:    allure_results/1ef96be7-358e-4d33-8c37-fa7f256a2f5b-container.json
	deleted:    allure_results/206c9c7e-7f75-4bbb-aeaa-e7a5d1b293e9-container.json
	deleted:    allure_results/25a4c4ca-e2d8-4aed-a593-d0fa84ec6f32-container.json
	new file:   allure_results/25d14896-b695-43de-b48b-06fe09305c18-container.json
	deleted:    allure_results/26690e3c-03a5-4939-8010-c619f4520455-container.json
	new file:   allure_results/26f978fd-d2c5-4f50-8cbb-58fedac1d52b-container.json
	deleted:    allure_results/277a2a9a-b1de-406f-9c5c-877f77e8eef7-container.json
	deleted:    allure_results/2b0aec79-f3bf-4f46-8ffc-40af9b28071f-container.json
	new file:   allure_results/2b89f8fe-ad17-4b36-90e3-5a0998c7bf3b-result.json
	deleted:    allure_results/2ba70434-0b59-4648-ac70-81d6462ab8bc-container.json
	new file:   allure_results/348ca2bd-86e4-42f1-bf9a-6e55c8193c8f-container.json
	deleted:    allure_results/3650e919-fb24-4a35-b852-f358b0eb0c74-container.json
	new file:   allure_results/36c10057-21d9-46fa-96e5-1a3c7ee24861-result.json
	new file:   allure_results/3a622f7a-5709-4e5a-9c14-adfc91d025ea-result.json
	new file:   allure_results/456807ea-f37e-4950-ab6a-e3c03c82ab5a-container.json
	deleted:    allure_results/473963c8-cda2-4e1b-b75d-b2460a23b0d6-container.json
	new file:   allure_results/4e20f7b8-ae17-4f13-a642-0acab64eea4a-attachment.txt
	new file:   allure_results/4fb63c5b-abcd-42da-963a-5247f8c212ee-container.json
	deleted:    allure_results/553fb0c9-7a62-48b4-b68a-e8e2232ba277-container.json
	new file:   allure_results/5a85eced-69e2-4460-bcc5-3523114ddff6-result.json
	deleted:    allure_results/5c27012f-6830-4563-acb0-274adf91c39a-container.json
	deleted:    allure_results/5f154c07-7c05-456f-913f-7d2372722bd0-result.json
	deleted:    allure_results/61b6a131-22c7-4820-bafb-48be65b70fc3-container.json
	deleted:    allure_results/650bc7e7-ad1f-4f4a-a7b1-85199b5f6cfe-container.json
	deleted:    allure_results/6a21acc8-3ccf-4775-80c0-4dcfe44dc396-result.json
	deleted:    allure_results/6b782ec5-4a3a-43c1-a7b8-3bb19d06d50c-result.json
	new file:   allure_results/6cdbeb35-de5c-4897-90ee-168357ccd966-container.json
	new file:   allure_results/70031247-a622-4fe8-bb0e-93477b5b7a35-container.json
	new file:   allure_results/7162814e-25b5-478a-9e8a-b70a9a479632-container.json
	new file:   allure_results/742cf5a3-4112-452b-8405-c8886936025b-container.json
	new file:   allure_results/75e6eac3-1b34-4dc9-8fe9-e744710c85e1-container.json
	deleted:    allure_results/7865499d-0716-4e16-a9a7-aa86339955af-container.json
	deleted:    allure_results/791a7738-fdb6-4bda-b76b-94a0c5f6aac8-container.json
	new file:   allure_results/7b28b9d9-5b6b-4d30-a1d4-ef189128d105-container.json
	deleted:    allure_results/7b3ba24a-f128-45f2-816b-ab50c7626999-container.json
	new file:   allure_results/80f786e7-b4ae-4d75-94aa-a0895dd3f10d-container.json
	new file:   allure_results/837b255d-34d9-4f49-a418-d1976ef1fe22-container.json
	new file:   allure_results/8b6b3869-846d-48e0-96ee-fe4bdf43fb40-container.json
	new file:   allure_results/8cd238c1-011c-4788-b636-e552d3ee4dd3-result.json
	new file:   allure_results/8d086e66-c1bb-464d-84b9-3557c36380d7-container.json
	new file:   allure_results/8ff222d7-fc29-4cb9-8633-ffb7d7afa7a5-container.json
	new file:   allure_results/91db8e11-3beb-48b5-970d-d07dfde06952-attachment.txt
	new file:   allure_results/93cfc780-b58d-4d10-bb9d-65f957c81123-result.json
	new file:   allure_results/93dc8945-93c2-4bac-8750-994b521d5cf1-container.json
	new file:   allure_results/97fe2e70-6310-4ac5-b4dd-709e7b168bfd-container.json
	deleted:    allure_results/9862a721-42fe-4cc9-9848-b0714bc21991-container.json
	deleted:    allure_results/99ce585c-5f1a-46c9-aa65-45117ccd630e-result.json
	deleted:    allure_results/9c70da65-e3fc-4ffa-bf40-49d8dc25784f-container.json
	new file:   allure_results/9cf7feae-bb52-40ec-93dc-1c3ff8dc8364-container.json
	new file:   allure_results/a13e23eb-c602-420a-af51-d438668aa25d-container.json
	deleted:    allure_results/a6204d5a-a252-43db-88c2-0c65c40f7b89-result.json
	new file:   allure_results/a7c297e8-f5d8-445b-ac4e-9f973f3c032a-container.json
	new file:   allure_results/afde1d37-e227-4639-9e2d-bcaaef0b4672-result.json
	new file:   allure_results/b0b8f602-2276-486c-9858-acfc459cfbc7-container.json
	deleted:    allure_results/b15a3024-96ab-4084-8469-3df26e788a86-container.json
	deleted:    allure_results/b4e63225-55e7-434b-a0fb-d1a594a11f75-result.json
	new file:   allure_results/b677c534-01d8-4dcb-8684-fe6f9483e6c4-container.json
	new file:   allure_results/bb23f29f-de0a-4d9d-8168-ed616ccce991-result.json
	new file:   allure_results/bbd8f577-3d65-4b97-8131-23ae6ae3b509-container.json
	new file:   allure_results/bc7edd71-db36-4708-868a-858a178b8414-result.json
	deleted:    allure_results/bdff784f-6bd5-463d-8f98-f0742ef563fe-container.json
	new file:   allure_results/bebe9564-6ccb-44ac-a955-4db843e1354e-container.json
	new file:   allure_results/bf9d3122-4ab2-4777-94fd-93cf94811ddc-container.json
	deleted:    allure_results/c4303dca-2cec-40ef-88a4-635351c90359-container.json
	new file:   allure_results/c6420a33-4427-4c45-91c3-e5fa134a3532-container.json
	deleted:    allure_results/c8bf5c30-b9de-45de-b262-2d65bf5330a6-container.json
	new file:   allure_results/caec7d92-654a-4106-ad54-6a0e0d84d110-container.json
	deleted:    allure_results/cc3e1048-3d0c-49d4-9a76-fd0c06cfd7c7-container.json
	new file:   allure_results/cfdf500f-6f1e-45a6-9bdd-82a9e5c73c46-attachment.txt
	new file:   allure_results/d001b42c-ed24-44c2-8c36-c776ee2410a9-container.json
	deleted:    allure_results/d35644c7-801c-48b8-8a36-50b592d7edd3-container.json
	new file:   allure_results/d3fe9d2e-1917-4679-aba7-dd2a60374a3e-result.json
	new file:   allure_results/e014b88a-a9df-4441-b4f9-1a721a14f6dd-result.json
	deleted:    allure_results/e0b6921e-6c8c-469e-8056-baa129161c0a-result.json
	new file:   allure_results/e12e7ac9-2815-4ee4-a271-76b4f3f1e7fc-container.json
	new file:   allure_results/e16c9d32-2ed0-4941-bbbc-3a0f5be20043-container.json
	deleted:    allure_results/ef784b5d-1877-4458-a070-1df970281138-container.json
	deleted:    allure_results/f471ccee-04db-4c5a-abd1-7a03a8a561f9-container.json
	deleted:    allure_results/f50b98ae-7b2d-4c1e-8f66-b2b79bef3c28-container.json
	new file:   allure_results/f6093cc7-5e14-47f1-82fa-6f179d1aab52-container.json
	new file:   allure_results/f6241b9c-eb80-4ac4-825c-21f443a63a89-result.json
	deleted:    allure_results/fa5a6bc4-cff9-4676-877a-04fa5a97a2bd-container.json
	deleted:    allure_results/fadc9b19-df43-4f5b-abd1-f2ddc72f9c9f-result.json
	new file:   allure_results/fe3696bd-49e2-4458-9e3b-66bcbab7d3a1-container.json
	deleted:    allure_results/feb7ec5b-9062-4729-b281-596ad16134e1-container.json
	deleted:    allure_results/ff0c27ec-5f56-4144-ac94-6726df8ec899-result.json
	modified:   tests/test_home_page.py
